shimmingtoolbox/shim/sequencer.py

```diff
@@ -936,46 +938,55 @@ def select_optimizer(method, unshimmed, affine, coils: ListCoil, opt_criteria, p
     return optimizer
 
 
-def _optimize(optimizer: Optimizer, nii_mask_anat, slices_anat, shimwise_bounds=None,
+@timeit
```

Comment:
```suggestion
@timeit
```
Calculating the time on a "private" function is fine in the PR but not to be displayed to the user